x-pack/plugins/ml/public/explorer/explorer_charts/explore_series.js

```diff
@@ -65,10 +65,5 @@ export function exploreSeries(series) {
     }
   });
 
-  let path = chrome.getBasePath();
-  path += '/app/ml#/timeseriesexplorer';
-  path += '?_g=' + _g;
-  path += '&_a=' + encodeURIComponent(_a);
-  window.open(path, '_blank');
-
+  return `${chrome.getBasePath()}/app/ml#/timeseriesexplorer?_g=${_g}&_a=${encodeURIComponent(_a)}`;
```

Comment:
Should this function continue to do the `window.open` part? Otherwise it might be better renaming it to something like 'getExploreSeriesLink'